doc/tutorials/using_windows_as_uos.rst

```diff
@@ -144,9 +72,9 @@ Create a raw disk
 
 Run these commands on the Service VM::
 
-   # swupd bundle-add kvm-host
-   # mkdir /root/img
-   # cd /root/img
+   # sudo apt-get install qemu
```

Comment:
Please change the prompts from `#` to `$` (on all lines)

Reply:
Done.